### ISSUE TITLE
Error when talking to Liam

### ISSUE THREAD

**Issue by DSeyka** [CleverRaven/Cataclysm-DDA#77075]
### Describe the bug

I'm getting the following error when talking to Liam:
```
 DEBUG    : Bad tag.  '<lets_go>' (0 - 8)

 FUNCTION : void parse_tags(std::string&, const talker&, const talker&, const dialogue&, const itype_id&)
 FILE     : src/npctalk.cpp
 LINE     : 2459
 VERSION  : cdda-experimental-2024-10-15-2059 e3ddac1
```
The dialogue seems to work as expected if error is ignored, but last dialogue option shows up as "????".
If picked, it ends the dialogue as expected.

### Attach save file

[Ellicott-trimmed.tar.gz](https://github.com/user-attachments/files/17386660/Ellicott-trimmed.tar.gz)

### Steps to reproduce

1. Talk to Liam
2. Get error

### Expected behavior

No error occurs

### Screenshots

![image](https://github.com/user-attachments/assets/ad407b58-b9df-4732-883c-698d118a06f3)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22631.4317 (23H2)
- Game Version: cdda-experimental-2024-10-15-2059 e3ddac1 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Stats Through Skills [StatsThroughSkills],
    Translate Complex Dialogue [translate_dialogue]
]

### Additional context

[debug.log](https://github.com/user-attachments/files/17386674/debug.log)

**Comment by AudBobb** [comment]
/confirm

Loaded up new world in 'Friends to the end' scenario with default settings, received same error when talking to Liam

**Comment by Rowmaster3** [comment]
Error is also present in refugee center. Same Bad Tag issue but with goodbyes instead of lets go (functionally the same i would imagine)

**Comment by dik1337** [comment]
I should have checked here first - #77080 is probably this same issue.

**Comment by GuardianDll** [comment]
Resolved by #76102
